### PR TITLE
[QPE-484] edit-mode interactions

### PR DESCRIPTION
**Pull request by ahmed-Bazzara**
JIRA ticket
https://tretton37.atlassian.net/secure/RapidBoard.jspa?rapidView=33&projectKey=QPE&modal=detail&selectedIssue=QPE-484

### CODE REVIEW

File: src/paint.jsx [hunk 1]
```diff
@@ -26,9 +26,10 @@ export default async function paint ($element, layout, component) {
     styling: { useSeparatorColumns }
   } = state;
 
+  const editmodeClass = component._inEditState ? 'edit-mode' : '';
```
Comment (muenva) [inline src/paint.jsx]: shouldn't we use `component.inEditState()` instead?

Comment (ahmed-Bazzara) [reply src/paint.jsx]: nope, the property name is with underscore.
not using it did not work

Comment (kristofferlind) [reply src/paint.jsx]: component.inEditState() would avoid the eslint warning and its not unusual for _ to denote a private variable, so the other version is probably less likely to break in the future.

Comment (muenva) [reply src/paint.jsx]: I was making the assumption based on another similar property updated by @Caele  in https://github.com/qlik-oss/QlikSense_Extension_2DimHeatmap/pull/41/files#diff-4f0be712c72e1f804a9ff53973e77a82R11

**Pull request closed without merging**